catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/impl/operations/DeleteOperations.java

```diff
@@ -473,4 +529,29 @@ private boolean isStorageAvailable(StorageProvider storageProvider) {
         }
         return true;
     }
+
+    private void logFailedQueryInfo(DeleteRequest deleteRequest, QueryResponse query) {
+        if (LOGGER.isDebugEnabled()) {
+            final String attributeName = deleteRequest.getAttributeName();
+            Set<String> queryResults = query.getResults()
+                    .stream()
+                    .map(Result::getMetacard)
+                    .map(m -> m.getAttribute(attributeName))
+                    .filter(Objects::nonNull)
+                    .map(Attribute::getValue)
+                    .filter(Objects::nonNull)
+                    .map(Object::toString)
+                    .collect(Collectors.toSet());
+
+            LOGGER.debug(
```

Comment:
I'm not in love with the three separate debug messages. However, I do recognize that you want to provide a decent amount of information that's well formatted. @stustison do you have any thoughts on this?